PusherMainView/PusherMainView/PusherStore.swift

```diff
@@ -248,6 +248,13 @@ extension PusherStore: PusherInteracting {
                 reduce(result: .failure(error, actionType))
             }
 
+        case .saveFileAs(let text, let fromViewController, let completion):
+            router.saveFileAs(from: fromViewController) { fileURL in
+                self.dispatch(actionType: .chooseFile)
```

Comment:
What does this action do?

Reply:
@imaginaris I renamed `chooseFile` to `enableSaveMenuItem `. Clearer I think.

Reply:
Yeah. Much clearer 🙂